ci/kokoro/docker/Dockerfile.fedora

```diff
@@ -36,15 +36,15 @@ RUN dnf makecache && dnf install -y \
     ncurses-term \
     openssl-devel \
     pkgconfig \
-    python \
+    python38 \
```

Comment:
Why python38? Why not just install both python and python3? Most people will not have python38 installed. The current version of `python3` is `3.7.5`, which is also the default in Fedora 31.

Reply:
I am happy to rely on 3.7.5. I didn't count on there being one. happy to change this :)